source/docs/hardware/hardware-basics/robot-battery.rst

```diff
@@ -0,0 +1,177 @@
+
+Screw terminal lugs are legal, but not recommended. If using screw terminal lugs, use the correct tip size screwdriver to tighten the terminal and check the terminal tightness periodically (at minimum, before each competition weekend).
+
+Battery Lead Lug Connection
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A #10 or M5 nut & bolt connect the battery lead lug to the battery tab.
+
+..warning:: The lug and tab must directly contact, copper to copper: do not put a washer of any kind separating them.
+
+Some batteries come with tab bolts in the package: they may be used, or replaced with stronger alloy steel bolts. It is a good idea to add a lock washer, such as a #10 star washer or a nordlock system, in addition to a nylon locking ("nylock") nut. Using a split ring washer will not add to joint strength or reliability and is not required even if one is provided with your battery.
+
+These connections must be very tight for reliability. Any movement of the lug while in operation may interrupt robot power, resulting in robot reboots and field disconnections lasting 30 seconds or more.
+
+This connection must also be completely covered for electrical safety; electrical tape will work, but heatshrink that fits over the entire connection is recommended. High shrink ratios (minimum 3:1, recommend 4:1) will make it easier to apply the heatshrink. Adhesive lined heat shrink is allowed, but not recommended. Be sure *all* the copper is covered - heat shrink may be "touched up" with electrical tape if some copper shows.
+
+.. image:: images/robot-battery/heatshrink.png
+  :alt: One terminal of an FRC battery fully covered in heatshrink.
+
+Battery Chargers
+^^^^^^^^^^^^^^^^^^^
+
+There are many good COTS "smart" battery chargers designed for 12V SLA batteries, rated for 6A or less per battery, with 'maintenance mode' features. Chargers rated over 6A are not allowed in FRC pits.
+
+Chargers used at competition are required to use Anderson SB connectors. Attaching a COTS SB connector battery lead to the charger leads using appropriately sized wire nuts or screw terminals is fast and simple (be sure to cover any exposed copper with heat shrink or electrical tape). SB Connector Contacts are also available for smaller wire sizes, if the team has crimping capability.
+
+..warning:: After attaching the SB, double check the charger polarities with a multimeter before plugging in the first battery.
+
+Some FRC vendors sell chargers with red SB50 connectors pre-attached.
+
+Battery Evaluation Tools
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Voltmeter** or **Multimeter**
+
+A voltage reading will give you a good snapshot of what the Voc (Voltage open circuit, or "float voltage") is. Anything from 13.0 to 14.0 may show up for charged batteries, but depends as much on the individual battery as it does on real state of charge. Below 13V, it's worth trying to charge it more.
```

Comment:
This method is not recommended might be good here.

Reply:
Reworked this section to move the voltmeter to after both the battery charger and DS Log